mongodbatlas/private_endpoints.go

```diff
@@ -44,6 +44,7 @@ type PrivateEndpointConnection struct {
 
 // InterfaceEndpointConnection represents MongoDB Interface Endpoint Connection.
 type InterfaceEndpointConnection struct {
+	ID                            string `json:"id,omitempty"`                            // Unique identifier of the private endpoint you created in your AWS VPC or Azure VNet.
```

Comment:
looks good just curious is the id really optional?

Reply:
It's required to pass in body request.